src/panels/config/tags/dialog-tag-detail.ts

```diff
@@ -191,13 +195,43 @@ class DialogTagDetail extends LitElement implements HassDialog {
     }
   }
 
+  private async _generateQR() {
+    let canvas = await QRCode.toCanvas(
```

Comment:
```suggestion
    const canvas = await QRCode.toCanvas(
```

Reply:
I cannot understand why eslint doesn't tell me about this when I run it locally for the life of me!